function func(params) {
    throw "erooooo"
    throw "erooooo"
    throw "erooooo"
    throw "erooooo"
    console.log("oi")
}

try {
    func()
} catch (error) {
    console.log(error)
}
